package.json

```diff
@@ -20,16 +20,16 @@
   "main": "./lib",
   "dependencies": {
     "buffer-writer": "2.0.0",
+    "connection-string": "^3.0.4",
```

Comment:
connection-string follows a different spec from libpq. I think a PostgreSQL driver should stick with the libpq format.

Reply:
I know.